caasp-istio-pilot-image/_service

```diff
@@ -0,0 +1,10 @@
+<services>
+    <service name="replace_using_package_version" mode="buildtime">
+        <param name="file">caasp-istio-pilot-image.kiwi</param>
+        <param name="regex">%%PKG_VERSION%%</param>
+        <param name="parse-version">patch</param>
+        <param name="package">istio-pilot</param>
```

Comment:
We are installing the `istio` package not istio-pilot. Is it correct? Not sure if `istio-pilot` is a dependency of istio. Anyway it not evident to me to see a package here that is not listed in `*.kiwi` XML.

Reply:
@davidcassany : Thats indeed a typo. We only need the `pilot-discovery` binary to be installed. But this is provided by the package `istio`. Thanks for catching it.